CalculatorExercises/1-CalculatorBasic/Calculator.java

```diff
@@ -0,0 +1,57 @@
+import java.util.*;
+import java.lang.*;
+
+public class Calculator {
+   
+    private static double result = 0;
+
+    public static double getResult() {
+        return result;
+    }
+
+    public static void calculation (StringBuilder input) {
+        
+        StringBuilder numberek = new StringBuilder("");
```

Comment:
All whitespaces are removed so we can assume that the number starts at input[1], so instead rewriting input to numberek you can do double number = Double.valueOf(input.substring(1))
https://www.tutorialspoint.com/java/java_string_substring.htm